.github/workflows/build.yaml

```diff
@@ -18,3 +18,15 @@ jobs:
       - run: cp .env-dist .env
       # Verify that the build (incl. type-checking) succeeds
       - run: npm run build
+        env:
+          UPLOAD_SENTRY_SOURCEMAPS: false
+        # Do not upload sourcemaps for non-release (tag) builds.
+        if: ${{ !startsWith(github.ref, 'refs/tags/') }}
```

Comment:
I think it's possible that this approach only works for `on: release`

Ref: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
Hm, I think we can either do that, or use `push`. I'd like to avoid duplicating all the steps if possible, I think https://docs.github.com/en/actions/using-workflows/reusing-workflows might be the way to do this?